src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/RegionalProcessingCenter.java

```diff
@@ -21,7 +21,8 @@ public RegionalProcessingCenter(@JsonProperty("faxNumber") String faxNumber,
                                     @JsonProperty("address3") String address3,
                                     @JsonProperty("postcode") String postcode,
                                     @JsonProperty("city") String city,
-                                    @JsonProperty("email") String email) {
+                                    @JsonProperty("email") String email,
+                                    @JsonProperty("listAssist") String listAssist){
```

Comment:
Is this the field that Jonathan was working on? If so I think this ought to be a boolean